### PR TITLE
Better error message for missing connection string

### LINKED ISSUE #599: Failed to load attached account
screen shot 2018-05-02 at 9 51 19 am

I'm not sure how my attached account got into this state, but the connection string was missing from keytar. I feel like we just need a better error message if the value returned by 'getPassword' [here](https://github.com/Microsoft/vscode-cosmosdb/blob/master/src/tree/AttachedAccountsTreeItem.ts#L239) is undefined/null/empty string

### PR DESCRIPTION
**Pull request by StephenWeatherford**
Fixes #599

### CODE REVIEW

File: src/mongo/tree/MongoAccountTreeItem.ts [hunk 1]
```diff
@@ -46,6 +46,10 @@ export class MongoAccountTreeItem implements IAzureParentTreeItem {
         try {
             let databases: IDatabaseInfo[];
 
+            if (!this.connectionString) {
+                throw new Error('Missing connection string');
```
Comment (EricJizbaMSFT) [inline src/mongo/tree/MongoAccountTreeItem.ts]: How about 'Failed to retrieve connection string.'

File: src/mongo/tree/MongoAccountTreeItem.ts [hunk 2]
```diff
@@ -46,6 +46,10 @@ export class MongoAccountTreeItem implements IAzureParentTreeItem {
         try {
```
Comment (EricJizbaMSFT) [inline src/mongo/tree/MongoAccountTreeItem.ts]: Technically, the same problem could happen for non-Mongo accounts

Comment (StephenWeatherford) [reply src/mongo/tree/MongoAccountTreeItem.ts]: For other accounts, you get this:
image

Fixing it to be nicer requires deeper changes than I think are worth the risk unless we get feedback that this is happening a lot.

Comment (EricJizbaMSFT) [reply src/mongo/tree/MongoAccountTreeItem.ts]: Fine with me. I think the effort for this fix accurately matched the priority of the bug

**Pull request merged**